src/components/CampusAudioTour.tsx

```
@@ -0,0 +1,201 @@
+import { useEffect, useState, useRef } from 'react';
```

ESLint

[error] Replace `'react'` with `"react"`

(prettier/prettier)